Networking/Sources/Extended/Remote/ProductVariationsRemote.swift

```diff
@@ -78,18 +78,23 @@ public class ProductVariationsRemote: Remote, ProductVariationsRemoteProtocol {
     /// - Returns: Variations for the provided parent product.
     public func loadVariationsForPointOfSale(for siteID: Int64,
                                              parentProductID: Int64,
-                                             pageNumber: Int = Default.pageNumber) async throws -> PagedItems<ProductVariation> {
+                                             pageNumber: Int = Default.pageNumber) async throws -> PagedItems<POSProductVariation> {
         let request = productVariationsRequest(for: siteID,
                                                productID: parentProductID,
                                                variationIDs: [],
                                                downloadable: false,
                                                status: .published,
                                                orderBy: .menuOrder,
                                                order: .ascending,
+                                               fields: POSProductVariation.requestFields,
```

Comment:
Inside `requestFields` we're passing `downloadable`,  but we're already using `downloadable: false` in the request. Could this be removed from one or the other? 

```
rest_route=/wc/v3/products/$PRODUCTID/variations
_fields=id,parent_id,attributes,image,price,regular_price,sale_price,on_sale,sku,global_unique_id,downloadable,manage_stock,stock_quantity,stock_status
```

Reply:
No – `downloadable: false` is a filter, but doesn't require that the field is included in the response. In general, we want server-side filtered to `downloadable: false`, but for some scanning requests, we don't want it filtered on the server, so we can show "could not add this to cart, because it's downloadable"